data/human/rim jobs.txt

```diff
@@ -100,9 +100,11 @@ mission "Han Sizer Month [0]"
 		distance 5 10
 		attributes "rim"
 		not attributes "station"
+	on enter
+		dialog `As you enter the system your passengers gather together to stare out the viewport. One of them pulls out a datapad and reads what sounds like a log entry by Han Sizer, describing the initial exploration of the system.`
 	on complete
 		payment 10000 170
-		dialog `Your passengers thank you for taking them on a journey around the Rim, and you collect your payment of <payment>.`
+		dialog `Your passengers thank you for taking them on a journey around the Rim, and you collect your payment of <payment>. As they depart you hear them talking about celebrating their trip with a party on Longjump.`
```

Comment:
What if we are on Longjump, though? The other dialogs have this same issue. It seems like for these to not be awkward, Longjump would have to not be a valid source or stopover.

Reply:
Might be able to use a branch here to check for the player's location and alter the text if it is on longjump. Would that be acceptable?

Reply:
Dialogs can't branch.

Reply:
Alright, sounds like a feature request, then.

Reply:
~~Use a substitution.~~

Reply:
substitution?

Reply:
Ah. https://github.com/endless-sky/endless-sky/wiki/CreatingSubstitutions

Reply:
I've got some substitutions now.